spec/features/user_creates_new_folders_spec.rb

```diff
@@ -5,57 +5,38 @@
     it "can see the folder they created" do
       user = User.create(username: "John Elway", password: "je", password_confirmation: "je", email: 'je@je.com', cellphone: '1234561234')
 
-      visit '/login'
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(user)
+      allow_any_instance_of(ApplicationController).to receive(:current_folder).and_return(user.root_folder)
 
```

Comment:
use this technique so we don't need to go through the login process for every feature test